change/@azure-msal-node-b1f6afd4-c1f6-408a-82a9-6d9e8a063035.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Merge branch 'dev' of github.com:AzureAD/microsoft-authentication-library-for-js into node_error_masking",
```

Comment:
Please fix with relevant comment about this PR.

Reply:
I think that the app developers need to have as much information as possible to make decisions about the server errors